packages/interactivity/src/directives.js

```diff
@@ -236,11 +239,14 @@ export default () => {
 			const defaultEntry = context.find(
 				( { suffix } ) => suffix === 'default'
 			);
-
 			// No change should be made if `defaultEntry` does not exist.
 			const contextStack = useMemo( () => {
 				if ( defaultEntry ) {
 					const { namespace, value } = defaultEntry;
+					// Check that the value is a JSON object. Send a console warning if not.
+					if ( SCRIPT_DEBUG && isPlainObject(value) ) {
+						console.warn('The value of data-wp-context in "%s" store must be a valid stringified JSON object.')
```

Comment:
I think the condition is off, there's a missing %s replacement, and the formatting looks off too. Maybe this would be good?

```suggestion
					if ( SCRIPT_DEBUG && ! isPlainObject( value ) ) {
						console.warn( 'Context values must be objects, found a value of type "%s" in the "%s" store.', typeof value, namespace );
```

Reply:
I thought I've changed it. I guess I messed with the push.